uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -280,7 +275,8 @@ private fun AppBar(
                         pagerState.animateScrollToPage(index)
                         tabLazyListStates.getValue(tab).animateScrollToItem(index = 0)
                     }
-                }
+                },
+                modifier = Modifier.zIndex(1f)
```

Comment:
I think it simpler to use `zIndex` instead of maintaining custom `ScrollableTabRow` 🤔

Reply:
I'm not sure, but is there a real indicator underneath this? 👀

Reply:
I didn't know that 👍 Thank you!!